src/main/java/spitter/web/services/AccountService.java

```diff
@@ -23,33 +24,33 @@
         users = new ArrayList<>();
         AccountModel account = new AccountModel();
-        account.setStrID("qwe");
-        account.setStrPass("qwe");
-        account.setStrName("asd");
-        account.setStrEmail("qwe@qwe.com");
+        account.setID("qwe");
+        account.setPass("qwe");
+        account.setName("asd");
+        account.setEmail("qwe@qwe.com");
         users.add(account);
-        courseRegisters = new ArrayList<CourseRegister>();
-        courseRegisters.add(new CourseRegister(account,new Course(1, "JavaScript", "JavaScript is a programming language used to make web pages interactive. ")));
-        courseRegisters.add(new CourseRegister(account, new Course(2, "HTML", "HTML is a computer language devised to allow website creation.")));
-        courseRegisters.add(new CourseRegister(account, new Course(3, "JQuery", "JQuery is a programming language used to make web pages interactive")));
-        courseRegisters.add(new CourseRegister(account, new Course(4, "CSS", "CSS is a computer language devised to allow website creation.")));
-        courseRegisters.get(0).setApproved(true);
-        courseRegisters.get(3).setApproved(true);
+        encrollments = new ArrayList<Encrollment>();
+        encrollments.add(new Encrollment(1,account,new Course(1, "JavaScript", "JavaScript is a programming language used to make web pages interactive. ")));
+        encrollments.add(new Encrollment(2,account, new Course(2, "HTML", "HTML is a computer language devised to allow website creation.")));
+        encrollments.add(new Encrollment(3,account, new Course(3, "JQuery", "JQuery is a programming language used to make web pages interactive")));
+        encrollments.add(new Encrollment(4,account, new Course(4, "CSS", "CSS is a computer language devised to allow website creation.")));
+        encrollments.get(0).setStatus(EncrollStatus.APPROVED);
+        encrollments.get(3).setStatus(EncrollStatus.APPROVED);
     }
 
     public Boolean login(AccountModel account) {
         //readListUserFromFile();
-        if (account.getStrID() != null && !account.getStrID().isEmpty()) {
+        if (account.getID() != null && !account.getID().isEmpty()) {
             for (AccountModel user : users) {
-                if (account.getStrID().equals(user.getStrID()) && account.getStrPass().equals(user.getStrPass())) {
-                    account.setStrName(user.getStrName());
-                    account.setdBirthDay(user.getdBirthDay());
-                    account.setStrEmail(user.getStrEmail());
-                    account.setStrGender(user.getStrGender());
+                if (account.getID().equals(user.getID()) && account.getPass().equals(user.getPass())) {
+                    account.setName(user.getName());
+                    account.setBirthDay(user.getBirthDay());
+                    account.setEmail(user.getEmail());
+                    account.setGender(user.getGender());
                     return true;
```

Comment:
We should have two models: one for `Login` page (username, password) and one for `User` details (full attributes)